transaction/std/src/transaction_builder.rs

```diff
@@ -38,6 +38,22 @@ impl TxOutputsOrdering for DefaultTxOutputsOrdering {
     }
 }
 
+/// Transaction output context is produced by add_output method
+/// Used for receipt creation
+#[allow(dead_code)]
```

Comment:
Why is this needed?

Reply:
Without that compilation fails due to the shared secret being dead code.